rust/tools/abacus-cli/src/subcommands/db_state.rs

```diff
@@ -44,41 +44,23 @@ impl DbStateCommand {
         Ok(())
     }
 
-    fn create_comitted_root_to_message_map(
-        &self,
-        db: &AbacusDB,
-    ) -> Result<HashMap<H256, Vec<CommittedMessage>>> {
-        let mut messages_by_committed_roots: HashMap<H256, Vec<CommittedMessage>> = HashMap::new();
+    fn create_comitted_root_to_message_map(&self, db: &AbacusDB) -> Result<Vec<CommittedMessage>> {
```

Comment:
is this name of the fn accurate? also curious why the change here

Reply:
Oh I just made the change to make the compiler happy but basically this command no longer makes any sense for us, so I just removed this for now, and we can re-add once we find it is useful for us specifically